benchmark/benchmarks/krausest/lib/components/Application.d.hbs.ts

```diff
@@ -0,0 +1,4 @@
+import type { SerializedTemplateWithLazyBlock } from '@glimmer/interfaces';
```

Comment:
There are two changes in all files in this directory:

1. Importing from `@glimmer/interfaces` rather than a relative path.
2. Using `.d.hbs.ts`, which is the new TypeScript default for `allowArbitraryExtensions` (see [the docs](https://www.typescriptlang.org/tsconfig/#allowArbitraryExtensions))